build.gradle.kts

```diff
@@ -135,7 +135,7 @@ subprojects {
 
 allprojects {
   group = "org.stellar.anchor-sdk"
-  version = "2.1.2"
+  version = "2.1.3"
```

Comment:
I don't think we bumped the version or updated the badges in `main`. Can we do that an merge `main` into `develop`?